api/src/main/java/net/starype/quiz/api/game/Question.java

```diff
@@ -0,0 +1,36 @@
+package net.starype.quiz.api.game;
+
+import net.starype.quiz.api.game.player.UUIDHolder;
+
+import java.util.List;
+
+public interface Question extends UUIDHolder {
+
+    List<QuestionTag> getTags();
+
+    default boolean isTagAttached(QuestionTag tag) {
+        return getTags().contains(tag);
+    }
+
+    void registerTag(QuestionTag tag);
+
+    void forgetTag(QuestionTag tag);
+
+    QuestionDifficulty getDifficulty();
+
+    String getQuestion();
+
+    String getCorrectAnswer();
```

Comment:
I think we talked about this, but the correctness of an answer should be determined by:
- a set of validity checks, that could instantly result in a 0 if the answer isn't even valid formatwise
- a correctness evaluation function, returning a value from 0 to 1. 

I would suggest adding one or two more interfaces to do that, and remove this method (since the correct answer should not be known outside this interface)

Reply:
The aim of ```getCorrectAnswer``` isn't to perform the validity check but just to get the correct answer (iE. printing the result)